README.md

```diff
@@ -3,7 +3,7 @@
 [![Tests](https://github.com/arduino/compile-sketches/workflows/Test%20Python%20code/badge.svg)](https://github.com/arduino/compile-sketches/actions?workflow=Test+Python+code)
 [![Lint](https://github.com/arduino/compile-sketches/workflows/Lint%20Python%20code/badge.svg)](https://github.com/arduino/compile-sketches/actions?workflow=Lint+Python+code)
 [![Spell Check](https://github.com/arduino/compile-sketches/workflows/Spell%20Check/badge.svg)](https://github.com/arduino/compile-sketches/actions?workflow=Spell+Check)
-[![codecov](https://codecov.io/gh/arduino/compile-sketches/branch/master/graph/badge.svg)](https://codecov.io/gh/arduino/compile-sketches)
+[![codecov](https://codecov.io/gh/arduino/compile-sketches/branch/main/graph/badge.svg?token=Uv6f1ebMZ4)](https://codecov.io/gh/arduino/compile-sketches)
```

Comment:
Are we leaking any information we should not be leaking with `token=xyz...`?

Reply:
The truth is that I have no clue what the deal is with this token and I didn't find any information about it after searching. This is the exact markup that Codecov provides:
https://app.codecov.io/gh/arduino/compile-sketches/settings/badge
It would be impossible to use the markup they provide in a manner that doesn't hardcode the token. Even though 

I have noticed that the previous badge markup we were using here and in other repos often just shows a broken image on the readme, which then comes back after you reload. I recently started using the officially recommended markup and haven't noticed the broken image thing happen in those repos, but it might be purely coincidental.

Reply:
Thank you for the clarification. Unfortunately I wasn't able to follow the link you provided (apparently I've got no permission) but I reckon the token is rather used for some identification than being able to do some mischief with it :shrug: